analyzers/src/SonarAnalyzer.CSharp/SyntaxTrackers/CSharpPropertyAccessTracker.cs

```diff
@@ -40,20 +40,19 @@ public class CSharpPropertyAccessTracker : PropertyAccessTracker<SyntaxKind>
 
         public CSharpPropertyAccessTracker(IAnalyzerConfiguration analyzerConfiguration, DiagnosticDescriptor rule) : base(analyzerConfiguration, rule) { }
 
+        public override object AssignedValue(PropertyAccessContext context) =>
```

Comment:
It backtracks constant values, so it could be renamed. But I'd keep it as `AssignedValue` because it could be value tracked from `x.Property = variable` back to it's constant source. Where from API user point of view, it provides whatever value we're able to track.